Tests/GameTest.cs

```diff
@@ -0,0 +1,37 @@
+﻿using System;
+using TicTacToe;
+using Xunit;
+
+namespace Tests.TicTacToe {
+    
+    public class GameTest {
+        
+        private Board board;
+        private Game game;
+
+        public GameTest() {
+            this.board = new Board();
+            IO io = new IO();
+            ValidateInput validateInput = new ValidateInput();
+            UI ui = new UI(io, validateInput);
+            this.game = new Game(this.board, ui);
+        }
+
+        [Theory]
+        [InlineData(0)]
+        [InlineData(1)]
+        [InlineData(2)]
+        [InlineData(3)]
+        public void ExpectMarkerToBePlaced(int move) {
+            string[] gameBoard = this.board.GameBoard;
```

Comment:
Should `GameTest` know that the board is stored as `string[]` in `Board`? If you wanted to change how the board were stored, how far would that change ripple throughout the system?